examples/vae/ss_vae_M2.py

```diff
@@ -197,6 +200,10 @@ def guide_classify(self, xs, ys=None):
         pass
 
     def model_sample(self, ys, batch_size=1):
+        """
```

Comment:
model_sample should be replaced with poutine magic

Reply:
unrelated to this PR

Reply:
this nit is unrelated to the change; no need to address it in this PR